### PR TITLE
update to 2.0 version

### PR DESCRIPTION
**Pull request by greenJVS**
Adopt 2.0 interface https://unpkg.com/browse/lightweight-charts@2.0.0/dist/typings.d.ts
Change version of js library to 2.0 for automatically update path version.

### CODE REVIEW

File: LightweightCharts.podspec [hunk 1]
```diff
@@ -19,9 +19,9 @@ Pod::Spec.new do |s|
       :script => 'LIBRARY=${TEMP_DIR}/lightweight-charts.js
       
 if [ "${CONFIGURATION}" = "Release" ]; then
-    curl -o $LIBRARY https://unpkg.com/lightweight-charts@1.2.1/dist/lightweight-charts.standalone.production.js
+    curl -o $LIBRARY https://unpkg.com/lightweight-charts@2.0/dist/lightweight-charts.standalone.production.js
```
Comment (timocov) [inline LightweightCharts.podspec]:
```suggestion
    curl -o $LIBRARY https://unpkg.com/lightweight-charts@2.0.0/dist/lightweight-charts.standalone.production.js
```

File: LightweightCharts.podspec [hunk 2]
```diff
@@ -19,9 +19,9 @@ Pod::Spec.new do |s|
...
 else
-    curl -o $LIBRARY https://unpkg.com/lightweight-charts@1.2.1/dist/lightweight-charts.standalone.development.js
+    curl -o $LIBRARY https://unpkg.com/lightweight-charts@2.0/dist/lightweight-charts.standalone.development.js
```
Comment (timocov) [inline LightweightCharts.podspec]:
```suggestion
    curl -o $LIBRARY https://unpkg.com/lightweight-charts@2.0.0/dist/lightweight-charts.standalone.development.js
```

File: LightweightCharts.podspec (hunk 1 above)
Comment (timocov) [reply LightweightCharts.podspec]: Can we use `s.version` here as variable? Or maybe define it once and use in all places?

Comment (greenJVS) [reply LightweightCharts.podspec]: This script copy to Xcode target just as plain text and we can not specify variables from other scope to it

**Pull request merged**